Makefile.am

```diff
@@ -38,13 +38,13 @@ EXTRA_DIST =
 CLEANFILES =
 DISTCLEANFILES =
 
-if ENABLE_DIST_CMAKEFILES
```

Comment:
I think the original issue is presence of CMakeList.txt and/or other *.cmake files in the repo root. This causes configure to think cmake is supported. Just `git rm` the offending files, and you can uncomment this piece (which will make it easier to regenerate and merge zproject improvements later).